packages/eas-cli/src/build/android/prepareJob.ts

```diff
@@ -106,3 +103,21 @@ export async function prepareJobAsync(
 
   return sanitizeJob(job);
 }
+
+function getCacheSettings(
```

Comment:
can you move that logic to eas-json package? The same behavior will be needed in www when we resolve to build a profile for builds using GitHub integration, so it would be good if the eas-json package already returned the profile without customPaths field.

I think it should be possible directly in joi schema to add some conditional logic that renames fields.

Reply:
Moved it to `eas-json`'s `EasJsonUtils`